<launch>
  <!-- <arg name="file_path" default="$(find position_saver_node)/output/position.txt" /> -->
  <!-- <arg name="topic_name" default="/djiros/rtk_position" /> -->
  <!-- <arg name="topic_name" default="/fixposition/navsatfix" /> -->

  <node pkg="serial_connect" type="RadarController" name="RadarController" output="screen">
    <param name="targetDistance" value="230" />
    <param name="distanceThreshold" value="80.0" />
    <param name="angular_z" value="0.08" />
    <param name="linear_x" value="0.3" />
    <param name="flag" value="1" />
    <param name="bufferSize" value="20" />
    <param name="threshold" value="50000.0" />
    <param name="log_flag" value="0" />
  </node>


</launch>
